src/test/java/seedu/address/logic/commands/ModelStub.java

```diff
@@ -0,0 +1,190 @@
+package seedu.address.logic.commands;
+
+import java.nio.file.Path;
+import java.util.function.Predicate;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyBuyerList;
+import seedu.address.model.ReadOnlyMeetUpList;
+import seedu.address.model.ReadOnlyPropertyList;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.buyer.Buyer;
+import seedu.address.model.meetup.MeetUp;
+import seedu.address.model.property.Property;
+
+/**
+ * A default model stub that have all of the methods failing.
+ */
+public class ModelStub implements Model {
```

Comment:
good that you extracted it into a new file by itself since so much code needs it